pom.xml

```diff
@@ -60,23 +81,17 @@
       <dependency>
         <groupId>org.reactivestreams</groupId>
         <artifactId>reactive-streams</artifactId>
-        <version>1.0.3</version>
+        <version>${reactive-streams.version}</version>
```

Comment:
personal preference: I'd only extract the versions that are common across a set of deps

Reply:
Given that we review versions quite often, I find it easier when they are consolidated in one section.